.eslintrc.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
       indent: ['error', 2, { SwitchCase: 1 }],
       quotes: ['error', 'single'],
       semi: ['error', 'always'],
-      'linebreak-style': ['error', 'unix'],
+      'linebreak-style': ['error', 'windows'],
```

Comment:
Please let's not do this - I think we always use unix files as standard in our repositories. This also would result into a big change: ✖ 13119 problems (13119 errors, 0 warnings)

Reply:
Changed it myself.